src/test/java/org/apache/commons/text/CharacterPredicatesTest.java

```diff
@@ -44,4 +44,77 @@ public void testDigits() {
         assertFalse(CharacterPredicates.DIGITS.test('.'));
         assertFalse(CharacterPredicates.DIGITS.test('L'));
     }
+
+    @Test
+    public void testArabicNumerals() {
```

Comment:
This whole test looks repetitive enough to benefit from [parameterized tests](https://github.com/junit-team/junit4/wiki/Parameterized-tests) or even possibly [theories](https://github.com/junit-team/junit4/wiki/Theories).